src/microbe_stage/MicrobeAI.cs

```diff
@@ -477,11 +477,15 @@ private void FleeFromPredators(Random random, Vector3 predatorLocation)
 
         microbe.LookAtPoint = targetPosition;
 
-        // If the predator is right on top of the microbe, there's a chance to try and swing with a pilus.
-        if (DistanceFromMe(predatorLocation) < 100.0f &&
-            RollCheck(SpeciesAggression, Constants.MAX_SPECIES_AGGRESSION, random))
+        if (DistanceFromMe(predatorLocation) < 100.0f)
         {
-            MoveWithRandomTurn(2.5f, 3.0f, random);
+            // If the predator is right on top of the microbe, there's a chance to try and swing with a pilus
+            if (RollCheck(SpeciesAggression, Constants.MAX_SPECIES_AGGRESSION, random))
+                MoveWithRandomTurn(2.5f, 3.0f, random);
+
+            // There's also a chance to jet away if we can
+            if (RollCheck(SpeciesFear, Constants.MAX_SPECIES_FEAR, random))
+                SecreteSlime();
```

Comment:
As you only used simple `if`s you may end up in situations where the AI both swings and jets... not sure if this should be kept.

If you want to simply shift to if/else, note that the order of conditions will be impactful, and so AI will try aggression before fear, resulting in higher rates of attacks than runaways for similar values of aggression and fear.

Reply:
Good spot. I've made this an else if and swapped the order, and it seems to create more linear paths for escaping microbes, which looks much better.